index.js

```diff
@@ -0,0 +1,70 @@
+const { ApolloServer, gql } = require('apollo-server');
+
+// This is a (sample) collection of books we'll be able to query
+// the GraphQL server for.  A more complete example might fetch
+// from an existing data source like a REST API or database.
+const Projects = [
+  {
+    name: 'Foo',
+    coverage: 100,
+    lastUpdated: 'yyyy-MM-dd'
+  },
+  {
+    name: 'Bar',
+    coverage: 75,
+    lastUpdated: 'yyyy-MM-dd'
+  },
+  {
+    name: 'Baz',
+    coverage: 50,
+    lastUpdated: 'yyyy-MM-dd'
+  },
+  {
+    name: 'Far',
+    coverage: 25,
+    lastUpdated: 'yyyy-MM-dd'
+  },
+  {
+    name: 'Boo',
+    coverage: 0,
+    lastUpdated: 'yyyy-MM-dd'
+  }
+];
+
+// Type definitions define the "shape" of your data and specify
+// which ways the data can be fetched from the GraphQL server.
+const typeDefs = gql`
+  # Comments in GraphQL are defined with the hash (#) symbol.
+
+  # This "Book" type can be used in other type declarations.
+  type Project {
+    name: String
+    coverage: Int
+    lastUpdated: String
```

Comment:
Would this not be some kind of DateTime?

Reply:
Graphql doesn't currently have a scalar(leaves on the graph) datetime object by default. 

We'll have to bring in a library to do that.  

Opened an issue here #2